frontend/www/js/omegaup/components/problem/CollectionList.vue

```diff
@@ -111,17 +111,30 @@ export default class CollectionList extends Vue {
   T = T;
   level = this.data.level;
 
-  get availableTags(): string[] {
-    let tags: Set<string> = new Set(
-      this.data.frequentTags.map((element) => element.alias),
+  get availableTags(): { alias: string; total: number }[] {
```

Comment:
una vez que se exponga el tipo, aquí se puede hacer

```suggestion
  get availableTags(): TagWithProblemCount[] {
```

igual en todos los demás lugares acá en TypeScript donde este tipo se pueda usar, en todos los archivos.